seq/src/main.rs

```diff
@@ -15,13 +15,14 @@ fn main() {
         let decimals = max_decimal_digits(&args);
         let padding = if matches.is_present("WIDTH") { Some(max_digits(&args)) } else { None };
         let (first, inc, last) = find_operands(&args);
+        let terminator = matches.value_of("TERMINATOR").unwrap_or("\n");
         let valid_range = (first <= last && inc > 0.0) || (first >= last && inc < 0.0);
         if valid_range {
             let seq = Seq::new(first, inc, last, decimals, separator, padding);
             for val in seq.into_iter() {
                 print!("{}", val);
             }
-            println!();
+            print!("\n{}", terminator);
```

Comment:
You're print a newline before printing the terminator. If I remember correctly (almost 2 years since I had used FreeBSD seq), the behavior would to just print the terminator